README.md

```diff
@@ -102,11 +102,6 @@ You can configure the client's network connection and key storage method with th
 | Parameter | Default | Description |
 | --------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | env       | `dev`   | Connect to the specified XMTP network environment. Valid values include `.dev`, `.production`, or `.local`. For important details about working with these environments, see [XMTP `production` and `dev` network environments](#xmtp-production-and-dev-network-environments).        |
-| apiUrl         | Undefined             | Manually specify an API URL to use. If specified, value of `env` will be ignored.                   |
-| keyStoreType   | `networkTopicStoreV1` | Persist the wallet's key bundle to the network, or use `static` to provide private keys manually.   |
-| codecs         | `[TextCodec]`         | Add codecs to support additional content types. To learn more, see [Different types of content](#different-types-of-content). |
-| maxContentSize | `100M`                | Maximum message content size in bytes.                                                              |
-| appVersion     | Undefined             | Add a client app version identifier that's included with API requests. For example, you can use the following format: `appVersion: APP_NAME + '/' + APP_VERSION`. Setting this value provides telemetry that shows which apps are using the XMTP client SDK. This information can help XMTP developers provide app support, especially around communicating important SDK updates, including deprecations and required upgrades.      |
```

Comment:
These features still aren't in this SDK. I've been waiting to hear if anyone wants them before implementing them.

Reply:
Ah, thank you for the clarification, @nakajima - makes sense!